recipe/conda_build_config.yaml

```diff
@@ -47,6 +60,25 @@ VERBOSE_AT:
   - V=1
 VERBOSE_CM:
   - VERBOSE=1
+
+# dual build configuration
+channel_sources:
+  - conda-forge,defaults
+  - conda-forge/label/new,defaults  # [linux or osx]
+
+channel_targets:
+  - conda-forge main
+  - conda-forge new                             # [linux or osx]
+
+docker_image:                                   # [linux or ppc64le]
+  - condaforge/linux-anvil                      # [linux]
+  - conda/c3i-linux64                           # [linux]
```

Comment:
I don't think we should/can use that image as is but rather need to apply some conda-forge-specific changes to it beforehand. xref: https://github.com/conda-forge/docker-images/pull/64#issuecomment-405881487